src/shared/components/post/post-listing.tsx

```diff
@@ -766,6 +774,21 @@ export class PostListing extends Component<PostListingProps, PostListingState> {
     this.setState({ showEdit: false });
   }
 
+  handleVideoLoadStart(e) {
+    const video = e.target as HTMLVideoElement;
+    const volume = localStorage.getItem("volume_level");
+    if (volume) video.volume = Number(volume);
+    else {
+      video.volume = 0;
+      localStorage.setItem("volume_level", "0");
+    }
+  }
+
+  handleVideoVolumeChange(e) {
+    const video = e.target as HTMLVideoElement;
+    localStorage.setItem("volume_level", video.volume.toString());
+  }
+
```

Comment:
These could be defined at the module level instead of as properties of the component.

Reply:
You mean somewhere like `src/shared/utils/media`?

Reply:
I mean in the same file they're already in, but in the outermost scope.

Reply:
With the recent changes, I brought back `linkEvent` as @dessalines suggested, should I still move?